web-marketplace/src/components/organisms/BuyModalOptions/BuyModalOptions.tsx

```diff
@@ -35,13 +35,12 @@ export const BuyModalOptions = ({
 }: Props) => {
   const cards = content?.cards ?? [];
   const setConnectWalletModalAtom = useSetAtom(connectWalletModalAtom);
-  const { loaded, wallet } = useWallet();
-  const connected = wallet?.address;
```

Comment:
the same logic is actually still there in `RegistryLayoutConnectWalletModal`, maybe we could fix this as part of this PR too?